bin/graftM

```diff
@@ -129,60 +132,59 @@ if __name__ == '__main__':
+    searching_options.add_argument('--search_and_align_only', action="store_true", help='Stop GraftM running after reads have been identified and aligned (i.e. no placement step, default=False)', default=False)
+    searching_options.add_argument('--search_only', action="store_true", help='Stop GraftM running after reads have been identified (i.e. no alignment or placement steps, default=False)', default=False)
+    searching_options.add_argument('--euk_check', action="store_true", help='Cross check identified reads using an 18S specific HMM to help filter out eukaryotic ribosomal reads (default=False)', default=False)
+    searching_options.add_argument('--search_method', choices=('hmmsearch','diamond'), help='Search method, either hmmsearch or DIAMOND (default=hmmsearch)', default='hmmsearch')
+    searching_options.add_argument('--maximum_range', type=int, help='Maximum range to use when searching for potentially linked reads (when searching contigs)', default=None)
+    searching_options.add_argument('--expand_search_contigs', nargs='+', help='Provide an alignment of the sample being searched. This alignment will initially be searched for full length genes, from which a sample specific HMM model will be created and used in the search step of graftM.')
     
     placement_options = graft_parser.add_argument_group('taxonomic assignment options')
-    placement_options.add_argument('--assignment_method', help='Taxonomic assignment method used (default=pplacer)', default=Run.PPLACER_TAXONOMIC_ASSIGNMENT, choices=(Run.PPLACER_TAXONOMIC_ASSIGNMENT,Run.DIAMOND_TAXONOMIC_ASSIGNMENT))
+    placement_options.add_argument('--assignment_method', help='Taxonomic assignment method, either pplacer (phylogenetic) or DIAMOND (pairwise). default = pplacer', default=Run.PPLACER_TAXONOMIC_ASSIGNMENT, choices=(Run.PPLACER_TAXONOMIC_ASSIGNMENT,Run.DIAMOND_TAXONOMIC_ASSIGNMENT))
     
     pplacer_options = graft_parser.add_argument_group('pplacer assignment options')
-    pplacer_options.add_argument('--placements_cutoff', metavar='confidence', help='Cutoff of placement confidence level (0.5 - 1), default=0.75', default=0.75)
-    pplacer_options.add_argument('--resolve_placements', action="store_true", help='Force taxonomy down to the best placement (default=False)', default=False)
-    pplacer_options.add_argument('--no_merge_reads', action="store_true", help='Merge forward and reverse read alignments for placement (default=False)', default=False)
+    pplacer_options.add_argument('--placements_cutoff', metavar='confidence', help='This flag allows you to change the likelihood cutoff for phylogenetic placement of reads (default=0.75).', default=0.75)
+    pplacer_options.add_argument('--resolve_placements', action="store_true", help='Ignore the placements cutoff and simply use the best placement assigned to the read. (default=False)', default=False)
```

Comment:
maybe "placement with the highest confidence". Or, is it "most highly resolved placement". I forget, but what would you say to removing this option? Noone uses it, right?